pkg/chains/signing.go

```diff
@@ -174,6 +179,9 @@ func (o *ObjectSigner) Sign(ctx context.Context, tektonObj objects.TektonObject)
 			if err != nil {
 				logger.Error(err)
 				continue
+			} else {
+				logger.Debugf("Recording count for signed messages for %v", obj)
```

Comment:
Are these debug message necessary? If I need to dig through the logs to debug an issue, these will mostly be noise. Maybe they should be logged at the trace level?